include/utility.hpp

```diff
@@ -517,6 +517,14 @@ template <class CB_CORE> struct TTableVar : public CBVar {
 
   TTableVar(TTableVar &&other) : CBVar() { std::swap(*this, other); }
 
+  TTableVar(std::initializer_list<std::pair<std::string_view, CBVar>> pairs)
+      : TTableVar() {
+    for (auto &kv : pairs) {
+      *payload.tableValue.api->tableAt(payload.tableValue, kv.first.data()) =
```

Comment:
This is wrong, it would need `cloneVar` (this is a C api so no magic happens)
I'd suggest using `(*this)[key]` instead.

Reply:
I'm confused with what the issue is. From what I can tell the `[]` operator does the same thing.